rails/7/gnarly.rb

```diff
@@ -345,12 +336,12 @@ def post_install_instructions
 end
 
 def format_ruby
-  run 'bin/rubocop -A --fail-level error'
+  run 'bundle exec rubocop -A --fail-level error'
 end
 
 def completion_notification
   puts ''
```

Comment:
Is this just so there's a newline? You could just put it at the start of your ascii_art and remove this.